src/modules/alwaysontop/AlwaysOnTop/FrameDrawer.cpp

```diff
@@ -183,13 +183,21 @@ D2D1_COLOR_F FrameDrawer::ConvertColor(COLORREF color)
 
 D2D1_ROUNDED_RECT FrameDrawer::ConvertRect(RECT rect, int thickness, int radius)
 {
-    auto d2d1Rect = D2D1::RectF((float)rect.left + thickness, (float)rect.top + thickness, (float)rect.right - thickness, (float)rect.bottom - thickness);
-    return D2D1::RoundedRect(d2d1Rect, (float)radius, (float)radius);
+    float halfThickness = thickness / 2.0f;
+    auto d2d1Rect = D2D1::RectF((float)rect.left + halfThickness + 1, 
+        (float)rect.top + halfThickness + 1, 
+        (float)rect.right - halfThickness - 1, 
+        (float)rect.bottom - halfThickness - 1);
+    return D2D1::RoundedRect(d2d1Rect, radius * 2.0f, radius * 2.0f);
```

Comment:
Not sure, but I would've thought this ought to be `radius = window.radius + halfThickness` ?

Reply:
No, halfThickness doesn't work here. Here is an example how does it looks like.
Scaling should be applied, but I think not here, so I moved it outside.
![image_2022-09-12_15-28-46](https://user-images.githubusercontent.com/8949536/189719295-86f6ae98-fb3f-4cc9-ad19-63ac94d8c586.png)